src/contents/en-gb/doc_xamarin.md

```diff
@@ -1,35 +1,36 @@
 
 ## Pre-requisites
 
 Before we start working on our first SAFE app, make sure you have the following tools installed to work with this tutorial.
 
 - **Visual Studio with Xamarin**: You need Visual Studio (any edition) to build our first SAFE Network mobile application. Follow the instructions [here](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/get-started/installation) to download and install visual studio with Xamarin.Forms for your Operating System.
-- **SafeAuthenticator**: Safe Authenticator mobile application will be required to initiate the authentication process with SAFE Network. Please follow installation instruction [here](https://github.com/maidsafe/safe_mobile#Installation).
+- **SafeAuthenticator**: Safe Authenticator mobile application will be required to initiate the authentication process with SAFE Network. Please follow installation instructions [here](https://github.com/maidsafe/safe_mobile#Installation).
+- Intermediate level knowledge about Xamarin.Forms.
 
-## Setup basic project
+## Setup a basic project
 
-- Create a new Project: Create a new cross-platform mobile app (Xamarin.Forms) project in visual studio.
-- Install MaidSafe.SafeApp package: Install the latest version of [MaidSafe.SafeApp](https://www.nuget.org/packages/MaidSafe.SafeApp/) package in all projects using NuGet package 
+- Create a new Project: Create a new **cross-platform mobile app (Xamarin.Forms)** project in visual studio.
+- Install MaidSafe.SafeApp package: Install the latest version of [MaidSafe.SafeApp](https://www.nuget.org/packages/MaidSafe.SafeApp/) package in all projects using NuGet package manager. 
 
 ## Using Mock Network
 
-We provide the mock feature with Safe APIs which can be used for fast app development and, also provides a safe space to perform test operations. 
+We provide the mock feature with SAFE APIs which can be used for fast app development and provides a safe space to perform test operations. 
 
-When we use the mock feature in an application, it does not communicate with the live network. Instead, it will interface with a local MockVault file in the system to simulate network operations, which is used to store and retrieve data. Let's see how we can set up and use a mock network.
+When we use the mock feature in an application, it does not communicate with the live network. Instead, it will interface with a local MockVault file in the system to simulate network operations, which is used to store and retrieve data. Let's see how we can set up and use a mock network:
 
-- Build conditional compilation symbols: Add `SAFE_APP_MOCK` flag for your project in "Properties > Build > Conditional compilation symbols".
+- Build conditional compilation symbols: Add `SAFE_APP_MOCK` flag for your project in "Properties > Build > Conditional compilation symbols". Add the flag in platform specific projects also.

```

Comment:
Build conditional compilation symbols: Add `SAFE_APP_MOCK` flag for your project in "**Properties** > **Build** > **Conditional compilation symbols**" `and` add the flag in `to` platform specific projects also.